backend/src/api/realms.rs

```diff
@@ -0,0 +1,95 @@
+use std::collections::HashMap;
+use futures::{stream::TryStreamExt, TryStream};
+
+use juniper::graphql_object;
+
+use super::Context;
+use deadpool_postgres::Pool;
```

Comment:
I don't have a hard opinion here, either, but I tend to sort "by usage." Here, the `Context` and `Pool` are needed because we need to talk to the db, Juniper is needed for the API definition and the first two are implementation details.

Reply:
Mh, yeah no, I rather want this changed :D